azure-pipelines/integrationtests.yml

```diff
@@ -1,27 +1,23 @@
 pool:
     name: ads-build-1es-hosted-pool
-    demands: 
+    demands:
     - ImageOverride -equals ADS-Linux_Image
 
 steps:
 
   - task: DockerInstaller@0
     displayName: Installing Docker
     inputs:
-      secureFile: 'testsettings.json'

```

Comment:
This isn't a valid property for the task and doesn't seem to be necessary with how our tests are currently running